modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/base/ReadOnlyUserStoreInitializerTestCase.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.identity.integration.test.base;
+
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.wso2.carbon.automation.engine.context.TestUserMode;
+import org.wso2.carbon.integration.common.admin.client.AuthenticatorClient;
+import org.wso2.carbon.integration.common.utils.mgt.ServerConfigurationManager;
+import org.wso2.identity.integration.common.clients.UserManagementClient;
+import org.wso2.identity.integration.common.utils.ISIntegrationTest;
+import org.wso2.identity.integration.test.util.Utils;
+import org.wso2.identity.integration.test.utils.ISTestUtils;
+
+import java.io.File;
+
+public class ReadOnlyUserStoreInitializerTestCase extends ISIntegrationTest {
```

Comment:
Add a class description. Address this at every other instance in this PR.

Reply:
Resolved in https://github.com/wso2/product-is/pull/7362/commits/d706fad3a498370b6d8ddf0208a9a091645d2975